server/auth/google.js

```diff
@@ -5,9 +5,6 @@ const {User} = require('../db/models')
 module.exports = router
 
 // retrieve google secret and client id
-const dotenv = require('dotenv')
-dotenv.config()
```

Comment:
looks like this may have gotten deleted out of your local machine's file. I don't know if that means you need to change the pull request, let's figure out the workflow for this :P